poetry/orb.yaml

```diff
@@ -17,15 +17,30 @@ jobs:
           Working directory for your package. Should point to a folder
           containing your pyproject.toml file.
         type: string
+      password:
+        default: $GEMFURY_TOKEN
```

Comment:
Thinking out loud, we should probably change these defaults to be noops at some point, or make them all unnecessary, or something like that. Maybe even let folks pass in an array of configs, since it's totally valid to depend on multiple source repos.

To be clear, this is 100% ok for now, just something for future thought.

Reply:
I have updated my change to make this a major release.